game-core/src/main/java/games/strategy/engine/chat/HeadlessChat.java

```diff
@@ -96,9 +89,7 @@ public void addMessageWithSound(final String message, final String from, final S
 
   private void addChatMessage(final String originalMessage, final String from) {
     final String message = Ascii.truncate(originalMessage, 200, "...");
-    final String time = "(" + TimeManager.getLocalizedTime() + ")";
-    final String prefix = showTime ? time + " " + from + ": " : from + ": ";
-    final String fullMessage = prefix + " " + message + "\n";
+    final String fullMessage = from + ": " + " " + message + "\n";
```

Comment:
Ok, this is what I don't quite get. Why is this feature being removed here?

Reply:
Probably better to keep it, the 'showTime' value is a constant:

```
 private boolean showTime = true;
```

Reply:
7b2979e0e